lib/core/services/implementations/feed_stream_updater_service_impl.dart

```diff
@@ -136,20 +142,47 @@ class FeedStreamUpdaterServiceImpl
         futures.add(_gettingFileData.getFileData(id: int.parse(fileId)));
       }
 
+      futures.add(
+        _gettingRatingList.getRatingList(
+          voteKeySigned: await _gettingVoteKeySigned.getVoteKeySigned(
+                authorId: post.authorID,
+                postId: post.id,
+              ) ??
+              '',
+        ),
+      );
+
       final data = await Future.wait(futures);
+
       final startPosFilesInData = postAuthor == null ? 1 : 0;
+      final endPosFilesInData =
+          startPosFilesInData + (post.files ?? []).length - 1;
       postAuthor ??= data.first;
 
       if (postAuthor == null) {
         return;
       }
 
       _lruCacheProfile.save(post.authorID, postAuthor);
+
+      List<FileData?> files = List<FileData?>.from(data.getRange(
+        startPosFilesInData,
+        startPosFilesInData + endPosFilesInData,
+      ));
+      List<FileData> filteredFiles = files
+          .where(
+            (element) => element != null,
+          )
+          .map(
+            (e) => e!,
+          )
+          .toList();
```

Comment:
```suggestion
      List<FileData> filteredFiles = files
          .where((element) => element != null)
          .map((e) => e!)
          .toList();
```

Может так лучше будет (только форматтером проверь)

Reply:
вроде если два слеша поставить после files, то разделит по-нормальному https://stackoverflow.com/a/74176437/14116050